### ISSUE TITLE
Build just times out

### ISSUE THREAD

**Issue by natashawong** [aws-amplify/amplify-console#844]
** Please describe which feature you have a question about? **
As Amplify tries to build my project, it stalls below (build logs below). I am unsure why it does this, no additional output comes after this.

** Provide additional details**
[log.txt](https://github.com/aws-amplify/amplify-console/files/4901641/log.txt)

**Comment by Athena96** [comment]
Hi @natashawong can you share your appId, amplify.yml (only if you're not using the default), and your package.json please. Thanks!

**Comment by natashawong** [comment]
Hi, thanks for the reply. 

AppID – not too sure where I could get that, but I'm guessing it's a part of this? 
arn:aws:amplify:ap-southeast-1:397309950213:apps/ds990smrznud7
If it's not it, let me know where to get it and I'll grab it for you. 

As for amplify.yml – I tried two versions (running backend and front end separately as well as using an npm package concurrently to run both at the same time) and both gave the same result where the build just stalls and times out. I've attached both. 

Package.json – I've attached it, I used the same package.json when running the two versions of the amplify.yml because it contained the settings to run either. 

[amplify.txt](https://github.com/aws-amplify/amplify-console/files/4906509/amplify.txt)
[amplify2.txt](https://github.com/aws-amplify/amplify-console/files/4906510/amplify2.txt)
[package.txt](https://github.com/aws-amplify/amplify-console/files/4906511/package.txt)
